addons/mrp/report/report_stock_rule.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="mrp_report_stock_rule" inherit_id="stock.report_stock_rule">
+        <xpath expr="//div[hasclass('o_report_stock_rule_rule')]/t" position="before">
+            <t t-if="rule[0].action == 'manufacture'">
+                <t t-if="rule[1] == 'destination'">
+                    <div class="fa fa-wrench fa-fw o_report_stock_rule_arrow" t-attf-style="color: #{color};"/>
+                </t>
+                <t t-if="rule[1] == 'origin'">
+                    <t t-call="stock.report_stock_rule_left_arrow"/>
+                </t>
+            </t>
+        </xpath>
+        <xpath expr="//div[hasclass('o_report_stock_rule_rule')]/t[last()]" position="after">
+            <t t-if="rule[0].action == 'manufacture'">
+                <t t-if="rule[1] == 'destination'">
+                    <t t-call="stock.report_stock_rule_right_arrow"/>
+                </t>
+                <t t-if="rule[1] == 'origin'">
+                    <div class="fa fa-wrench fa-fw o_report_stock_rule_arrow" t-attf-style="color: #{color};"/>
+                </t>
+            </t>
+        </xpath>
+        <xpath expr="//div[hasclass('o_report_stock_rule_symbol_legend')]/table/tbody" position="inside">
+            <tr>
+                <td><p>Manufacture</p></td>
```

Comment:
wrapping the content of a td in a p is weird to me
if you absolutely want to wrap, use a span